### PR TITLE
i18n(it): translate docs/contribute.mdx

### PR DESCRIPTION
**Pull request by clemenzi**
#### What kind of changes does this PR include?

- Translated content

#### Description

translated `contribute.mdx` in Italian

### CODE REVIEW

File: src/content/docs/it/contribute.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Ogni PR, specialmente le PR di traduzione, ha bisogno di revisioni! La revisione delle PR e il rilascio di commenti, suggerimenti o un approvazione con "LGTM!" ("Sembra buono per me!") è un ottimo modo per iniziare in qualsiasi repository e per apprendere di più su Astro.
+
+Abbiamo anche una comunità su [Discord](https://astro.build/chat) molto attiva! Valorizziamo i contributi di coloro che accolgono i nuovi membri, rispondono alle domande di supporto e condividono ciò che hanno costruito con e per Astro! Oltre ai contributi tradizionali su GitHub, Astro riconosce e supporta i membri della comunità che interagiscono con la nostra comunità, condividono Astro in post di blog, video e talk alle conferenze e contribuiscono al mantenimento della salute della nostra comunità.
+
+## Contribuire alla Documentazione
+
+Abbiamo diverse guide con informazioni su come iniziare a contribuire alla Documentazione di Astro!
+
+- [Guida Principale per la Contribuzione](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - Per favore consulta questa prima!
```
Comment (csarnataro) [inline src/content/docs/it/contribute.mdx:46]: I would rephrase this sentence as "Prima di iniziare, consulta questa guida." or something similar, without exclamation mark and possibly without "per favore".

File: src/content/docs/it/contribute.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: Contribuisci ad Astro
+description: Come partecipare e contribuire ad Astro.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Siamo aperti a qualsiasi tipo di contributo che sia grande o piccolo. In quanto progetto open-source, crediamo nel ritorno ai nostri collaboratori. Siamo felici di aiutarvi con indicazioni sulle PR, sulla scrittura tecnica e sulla trasformazione di qualsiasi idea in realtà.
```
Comment (csarnataro) [inline src/content/docs/it/contribute.mdx:10]: The sentence "crediamo nel ritorno" sounds weird to me. What about "cerchiamo di supportare attivamente i nostri collaboratori" or something similar?

File: src/content/docs/it/contribute.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Modi per Contribuire
+
+### Repository del progetto
+
+Ci sono molti modi per contribuire al Progetto Astro! Ogni repository di Astro ha un README con un link a un file `CONTRIBUTING.md` nel codice del progetto.
+
+Visita il [profilo GitHub di Astro](https://github.com/withastro) per trovare i repository relativi a:
+
+- Il [codice principale di Astro](https://github.com/withastro/astro), inclusi le integrazioni ufficiali e i modelli di avvio.
```
Comment (csarnataro) [inline src/content/docs/it/contribute.mdx:22]: I'm not sure a reader will understand what a "modello di avvio" is, just by reading this sentence. I would either keep the english term "starter template" or maybe "template di avvio". In my opinion, the term "template" is widely used in the developers' community.
What do you think?

**Review by csarnataro — COMMENTED** [review]
Very good translation, I just left a couple of comments/questions.

File: src/content/docs/it/contribute.mdx (hunk 3 above)
Comment (csarnataro) [reply src/content/docs/it/contribute.mdx:22]: Or "template di creazione"

File: src/content/docs/it/contribute.mdx (hunk 1 above)
Comment (vclemenzi) [reply src/content/docs/it/contribute.mdx:46]: Thanks for the correction! I'm preparing a commit with the changes

File: src/content/docs/it/contribute.mdx (hunk 2 above)
Comment (vclemenzi) [reply src/content/docs/it/contribute.mdx:10]: Thanks for the correction! I'm preparing a commit with the changes

File: src/content/docs/it/contribute.mdx (hunk 3 above)
Comment (vclemenzi) [reply src/content/docs/it/contribute.mdx:22]: Actually, it doesn't sound very good. Maybe it would be better to use a different term. I could use 'template' as the word, but if anyone has other ideas, I would be happy to receive them.

File: src/content/docs/it/contribute.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Ogni PR, specialmente le PR di traduzione, ha bisogno di revisioni! La revisione delle PR e il rilascio di commenti, suggerimenti o un approvazione con "LGTM!" ("Sembra buono per me!") è un ottimo modo per iniziare in qualsiasi repository e per apprendere di più su Astro.
+
+Abbiamo anche una comunità su [Discord](https://astro.build/chat) molto attiva! Valorizziamo i contributi di coloro che accolgono i nuovi membri, rispondono alle domande di supporto e condividono ciò che hanno costruito con e per Astro! Oltre ai contributi tradizionali su GitHub, Astro riconosce e supporta i membri della comunità che interagiscono con la nostra comunità, condividono Astro in post di blog, video e talk alle conferenze e contribuiscono al mantenimento della salute della nostra comunità.
+
+## Contribuire alla Documentazione
+
+Abbiamo diverse guide con informazioni su come iniziare a contribuire alla Documentazione di Astro!
+
+- [Guida Principale per la Contribuzione](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - Prima di iniziare, consulta questa prima!
```
Comment (csarnataro) [inline src/content/docs/it/contribute.mdx:46]: I think the last part of this sentence should read "consulta questa guida", not "questa prima".
Moreover I would remove the exclamation mark.

**Review by csarnataro — COMMENTED** [review]
Thanks for your updates. I think there's one last thing left to fix, I left a comment.

File: src/content/docs/it/contribute.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- [Guida alla Scrittura](https://github.com/withastro/docs/blob/main/contributor-guides/writing-and-style-guide.md) - Suggerimenti per scrivere nello stile della Documentazione di Astro.
+- [Guida alla Traduzione](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md) - Informazioni su come aiutare a tradurre la Documentazione di Astro nella tua lingua.
+- [Guida alle Ricette](https://github.com/withastro/docs/blob/main/contributor-guides/submitting-a-recipe.md) - Scopri come scrivere e contribuire con una ricetta.
```
Comment (csarnataro) [inline src/content/docs/it/contribute.mdx:49]: Elsewhere we have already used "soluzioni" for "recipes' (see https://discord.com/channels/830184174198718474/1125854416956825730/1138561951291682876). I would stick with "soluzioni", in order to be consistent everywhere.

**Review by csarnataro — COMMENTED** [review]
I left another final (i hope) comment about the term "recipe"

**Review by pietrodev07 — APPROVED** [review]
Very good translation!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀
